src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1431,6 +1431,17 @@
     /**
      * A right curly brace (<code>}</code>).
      *
+     * <p>For example:</p>
+     * <pre>
+     * <code>} // -Right Curly Brace</code>
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --RCURLY -&gt; }
+     * `--SINGLE_LINE_COMMENT -&gt; //
```

Comment:
Please use CLI output as is with all spaces.
Please make example with empty method body `void foo(){}`

Reply:
@romani Fixed it. I have also made the required changes in the PR description.

Reply:
code should be `void foo(){}` , strictly this. 

printed ast only for this line.

Reply:
Done. Are the changes fine?

Reply:
No, please keep in ast whole method definition. Users need to see context where this token might be used.

Please update PR description.

Reply:
@romani Done! I have also updated the PR description.